### PR TITLE
Update trees.py

### PR DESCRIPTION
**Pull request by hyanwong**

### CODE REVIEW

File: python/tskit/trees.py [hunk 1]
```diff
@@ -2630,6 +2630,47 @@ def mean_descendants(self, reference_sets):
...
+
+        .. note:: The reference sets need not include all the samples, hence the most
+            recent common ancestral node of the reference sets, :math:`a`, need not be
+            the immediate ancestor of the focal node. If the reference sets only comprise
+            sequences from relatively distant individuals, the GNN statistic may end up
+            as a measure of comparatively distant ancestry, even for tree sequences that
+            contain many closely related individuals.
+
+        :param iterable focal: A list of :math:`n` nodes whose GNNs should be calculated.
+        :param iterable reference sets: A list of :math:`m` lists of node IDs.
```
Comment (jeromekelleher) [inline python/tskit/trees.py]: should be reference_sets

File: python/tskit/trees.py [hunk 2]
```diff
@@ -2630,6 +2630,47 @@ def mean_descendants(self, reference_sets):
...
+        :return: An :math:`n`  by :math:`m` array of focal nodes by GNN proportions.
+            Every focal node corresponds to a row. The numbers in each
+            row corresponding to the GNN proportion for each of the passed-in reference
+            sets. Rows therefore sum to one.
+        :rtype: numpy.ndarray
+
```
Comment (jeromekelleher) [inline python/tskit/trees.py]: Blank line at the end of the docstring might confuse downstream tools.

**Comment by jeromekelleher** [comment]
Can you fix these minor issues and rebase please @hyanwong? (The branch is a bit behind master now)

**Comment by hyanwong** [comment]
Oh, sorry - didn't see this. Will do.

File: python/tskit/trees.py (hunk 1 above)
Comment (hyanwong) [reply python/tskit/trees.py]: Done, and also corrected it in the docs for the function above, where I accidentally copied it from

**Comment by hyanwong** [comment]
NB, I think this documentation is also compatible to the suggestion in https://github.com/tskit-dev/tskit/issues/148

**Comment by jeromekelleher** [comment]
Looks great, thanks @hyanwong.

**Pull request merged**